doc/introduction/tutorials.rst

```diff
@@ -124,7 +123,7 @@ Optimization and machine learning
 
     /tutorials/tutorial_machine_learning
     /gallery/minimizing_correlations/minimizing_correlations
-    /gallery/state_learner/StateLearning
+    ../tutorials_apps/run_state_learner
```

Comment:
Since we're moving the tutorials out of this repo soon, I wouldn't worry about it :) We can build up the new structure from scratch in the applications repo.